charts/bulk-scan-processor/values.aat.template.yaml

```diff
@@ -15,3 +15,13 @@ java:
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
     PROCESS_PAYMENTS_ENABLED: "true"
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    BULK_SCANNING_DB_USER_NAME: bulk_scanner@bulk-scan-processor-aat
```

Comment:
is this ok to be here?

Reply:
yup draft store has it too so thought it is fine